device/cuda/include/traccc/cuda/seeding2/kernels/kd_tree_kernel.hpp

```diff
@@ -0,0 +1,24 @@
+/** TRACCC library, part of the ACTS project (R&D line)
```

Comment:
In principle yes, I was in two minds about this because generalized accelerator structure construction code may be useful elsewhere, but we can put it in a private header also.